tests/test-addons.py

```diff
@@ -480,3 +481,22 @@ def test_sosivio(self):
         validate_sosivio()
         print("Disabling sosivio")
         microk8s_disable("sosivio")
+
+    @pytest.mark.skipif(
+        platform.machine() != "x86_64",
+        reason="Ondat tests are only relevant in x86 architectures",
+    )
+    @pytest.mark.skipif(
```

Comment:
Could you please not skip this test UNDER_TIME_PRESURE? For now that the addon is new lets always test.

Reply:
Sure, I get to adding this test once I have confirmed that it works in my local environment.

Reply:
@ktsakalozos I have gone ahead and tested the addon that does not skip the UNDER_TIME_PRESURE test and it works. I have gone ahead and removed this exception for Ondat.